brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Off-Canvas - Changelog click open in new tab.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/bd9b05e6-1105-491d-9bfa-175a00d2cefd?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/X6uEzXBL

### Types of changes




- Improvement.

### How has this been tested?

- Go to the admin dashboard click on the off-Canvas menu.
- Click on any changelog and check it is opening in new tab or not.

### Checklist:
